docs/guides/tutorials/getting_started/connect_to_data.rst

```diff
@@ -55,7 +64,7 @@ Configuring Datasources
 When you completed those last few steps in ``great_expectations init``, you told Great Expectations that:
 
 1. You want to create a new Datasource called ``my_postgres_db``.
-2. You want to use SqlAlchemy to evaluate your Expectations, hence ``data_asset_type.class_name = SqlAlchemyDataset``. Your database configuration tells the SqlAlchemyDataset to use a Postgresql-specific driver.
+2. You want to use SqlAlchemy to evaluate your Expectations, hence ``data_asset_type.class_name = SqlAlchemyDataset``. Your database configuration tells the SqlAlchemyDataset to use a Postgres-specific driver.
```

Comment:
My head just exploded!! :) I trust your research, but I did think postgresql was more correct.

Reply:
Could be either way (officially PostgreSQL but Postgres is also accepted), but I've used Postgres in another part. No strong opinions, don't think we need to really standardize, I just noticed this one.